betelgeuse/parser.py

```diff
@@ -187,3 +187,31 @@ def parse_docstring(docstring=None):
             field_value = output
             fields_dict[field_name] = field_value
     return fields_dict
+
+
+def parse_markers(all_markers=None, config=None):
+    """Parse the markers."""
```

Comment:
could you please expand on what the function actually does, what it accepts and what it is supposed to return?

Reply:
Explained !